guide/sequelize/currency.md

```diff
@@ -266,7 +266,7 @@ Reflect.defineProperty(currency, 'getBalance', {
 });
 ```
 
-We're defining an `.add()` method to our currency collection. It'll be used quite frequently, so having a method for it makes our lives easier. We'll also add a `.getBalance()` method so that we'll always get a number.
+We're defining an `.add()` method to our currency collection. We'll use it quite frequently, so having a method for it makes our lives easier. We'll also add a `.getBalance()` method so that we'll always get a number.
```

Comment:
"It'll be used" is fine

Reply:
"be used" is passive, which we're trying to avoid.